cmd/ocitools/generate.go

```diff
@@ -83,19 +87,30 @@ var generateCommand = cli.Command{
 			return err
 		}
 
+		specgen.ExportOpts.Seccomp = context.Bool("seccomp-only")
+
 		if context.IsSet("output") {
-			output := context.String("output")
-			err = specgen.SaveToFile(output)
+			specgen.ExportOpts.Output = context.String("output")
 		} else {
-			err = specgen.Save(os.Stdout)
+			specgen.ExportOpts.Output = "STDOUT"
```

Comment:
Without having a function like `exportConfig` you would have to have the check for stdout/path inside both the if and the else, no? Doesn't having that relegated to one function call seem cleaner?